eth/config.go

```diff
@@ -202,10 +202,7 @@ type Config struct {
 	EVMCallTimeOut time.Duration
 
 	// Quorum
-	EnableMultitenancy bool `toml:"-"`
-
-	// Quorum
-	SaveRevertReason bool `toml:"-"`
+	QuorumChainConfig core.QuorumChainConfig `toml:"-"`
```

Comment:
I think with
```
type Config struct {
    *core.QuorumChainConfig // or core.QuorumChainConfig
}
```
We would shorten the references to the config. E.g.: `b.eth.config.QuorumChainConfig.MultiTenantEnabled()` would now become `b.eth.config.MultiTenantEnabled()`